studio/migrations/projects/0001_initial.py

```diff
@@ -168,4 +171,24 @@ class Migration(migrations.Migration):
             name='project',
             field=models.ForeignKey(null=True, on_delete=django.db.models.deletion.CASCADE, related_name='ba_project', to='projects.project'),
         ),
+        migrations.AddField(
+            model_name='environment',
+            name='public',
+            field=models.BooleanField(default=False),
+        ),
+        migrations.AddField(
+            model_name='projecttemplate',
+            name='enabled',
+            field=models.BooleanField(default=True),
+        ),
+        migrations.AlterField(
+            model_name='environment',
+            name='project',
+            field=models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.CASCADE, to='projects.project'),
+        ),
+        migrations.AlterField(
+            model_name='environment',
+            name='slug',
+            field=models.CharField(blank=True, max_length=100, null=True),
+        ),
```

Comment:
I wonder if this will work on prod db

Reply:
I think this is unchanged migrations. Why wouldn't it work you think?